sdk/attestation/attestation/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
should this extend our default client config for typescript?

Reply:
Agreed. Manually modified for this PR. Filed https://github.com/Azure/autorest.typescript/issues/809